pyroomacoustics/tests/test_whitening.py

```diff
@@ -0,0 +1,36 @@
+# @version: 1.0  2018, Juan Azcarreta
+from unittest import TestCase
+
+import numpy as np
+import pyroomacoustics as pra
+from scipy.stats import ortho_group
+
+class TestWhitening(TestCase):
+
+    def test_whitening(self):
+        # Create multivariate distribution
+        mean = [0, 0]               # zero mean
+        covx = [[100, 5], [5, 2]]   # diagonal covariance positive-semidefinite, e.g., covx = [[100, 5], [5, 2]]
+        samples = 5000
+
+        # Test the input
+        assert np.all(np.linalg.eigvalsh(covx) >= 0), "Covariance matrix is not positive-semidefinite"
+        dimensions = len(covx)      # should be equal to two
+        assert dimensions == 2, "The number of dimensions must be equal to two"

```

Comment:
I don't think this `assert` and the one above are necessary. It's not really testing the whitening and the conditions should be satisfied as you're manually setting the values.